docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+
+```jldoctest dataframe
+julia> german[!, [:Sex]]
+1000×1 DataFrame
+  Row │ Sex
+      │ String
+──────┼────────
+    1 │ male
+    2 │ female
+    3 │ male
+    4 │ male
+    5 │ male
+    6 │ male
+    7 │ male
+    8 │ male
+  ⋮   │   ⋮
+  994 │ male
+  995 │ male
+  996 │ female
+  997 │ male
+  998 │ male
+  999 │ male
+ 1000 │ male
+985 rows omitted
+
+julia> german[!, [:Sex]] == german[:, [:Sex]]
+true
+
+julia> german[!, :Sex]
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+
+julia> german[!, :Sex] == german[:, :Sex]
+true
+```
+
+In the first case, `[:Sex]` is a vector, indicating that the resulting object should be a `DataFrame`.
+On the other hand, `:Sex` is a single symbol, indicating that a single column vector should be extracted.
+Note that in the first case a vector is required to be passed (not just any iterable),
+so e.g. `german[:, (:Age, :Sex)]` is not allowed, but `german[:, [:Age, :Sex]]` is valid.
+
+### Most elementary `get` and `set` operations
+
+Given the DataFrame `german` earlier we have created, here are various ways to grab one of its columns as vector:
+```jldoctest dataframe
+julia> german
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0     67  male        2  own      NA               little          ⋯
+    2 │     1     22  female      2  own      little           moderate
+    3 │     2     49  male        1  own      little           NA
+    4 │     3     45  male        2  free     little           little
+    5 │     4     53  male        2  free     little           little          ⋯
+    6 │     5     35  male        1  free     NA               NA
+    7 │     6     53  male        2  own      quite rich       NA
+    8 │     7     35  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+
+julia> german.Age, german[!, 2], german[!, :Age] # all get the vector stored in out DataFrame without copying it
```

Comment:
do not compress + why do you repeat what you already explained (maybe it is OK, but just please explain to me)

Reply:
In the previous example I had explained using `col` variable (ie, `:Age`). here I have used indexes

Reply:
OK, but do not compress + explain the difference